Cargo.toml

```diff
@@ -10,8 +10,8 @@ edition = "2018"
 kelvin = "0.19.0"
 nstack = "0.5"
 lazy_static = "1.3.0"
-hades252 = { git = "https://github.com/dusk-network/hades252", tag = "v0.7.0" }
-dusk-plonk = "0.2.8"
+hades252 = { git = "https://github.com/dusk-network/hades252", tag = "v0.8.0" }
+dusk-plonk = {version = "o.2.11", features = ["trace-print"]}
```

Comment:
Change the `o` into a `0`, I imagine this gives errors otherwise.

Reply:
Ohh Didn't see it. Ty